tf-build-tools/tf-maven-plugin/src/test/java/com/deliveredtechnologies/maven/terraform/mojo/TerraformWrapperTest.java

```diff
@@ -0,0 +1,4 @@
+package com.deliveredtechnologies.maven.terraform.mojo;
```

Comment:
Think about how you might test this.

Reply:
I will speak to you at 2 PM for some guideance

Reply:
@Clayton7510  I pushed up the WrapperTest.java plus the changes you showed me to Wrapper.java.  Please let me know if any more changes need to be made. Thanks.